interp/gta.go

```diff
@@ -187,11 +187,14 @@ func (interp *Interpreter) gta(root *node, rpath, importPath string) ([]*node, e
 			}
 			// Try to import a binary package first, or a source package
 			var pkgName string
-			if interp.binPkg[ipath] != nil {
+			if pkg := interp.binPkg[ipath]; pkg != nil {
 				switch name {
 				case "_": // no import of symbols
 				case ".": // import symbols in current scope
-					for n, v := range interp.binPkg[ipath] {
+					for n, v := range pkg {
+						if n[0] == '.' {
+							continue
+						}
```

Comment:
I do not understand that part. what is it for?

Reply:
It's a leftover from previous attempt, and should be removed. Thanks for noticing it.